stable/gemini/README.md.gotmpl

```diff
@@ -0,0 +1,20 @@
+<div align="center">
```

Comment:
I think we typically like to have install instructions in the README, don't we?

Reply:
👍 added for goldilocks too

Reply:
Thanks!